haskell-tree-sitter.cabal

```diff
@@ -1,31 +1,30 @@
+cabal-version:       2.2
```

Comment:
Is this required for any of the changes here, or is it just a good idea in general?